WalletWasabi.Tests/UnitTests/SmartLabelTests.cs

```diff
@@ -125,6 +163,10 @@ public void CaseSensitivityTests()
 		smartLabel = new SmartLabel("bAr, FOO, Buz");
 		smartLabelToCheck = new SmartLabel("buZ, BaR, fOo");
 		stringLabelToCheck = "buZ, BaR, fOo";
+		Assert.False(smartLabel.Equals(smartLabelToCheck));
+		Assert.False(smartLabel.Equals(stringLabelToCheck));
+		Assert.True(0 == smartLabel.CompareTo(smartLabelToCheck));
```

Comment:
@wieslawsoltes, the default comparison is handled by tests here now.

Reply:
Great